cron/hourly/git_maintenance_tasks_lock

```diff
@@ -4,4 +4,5 @@ set -e
 # Run git_maintenance_tasks only if not running already
 DIR="$(readlink -e "$(dirname $0)/..")"
 mkdir -p ~/lmms-repo
-flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock "$DIR/git_maintenance_tasks"
+flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock \
+	"$DIR/git_maintenance_tasks_retry"
```

Comment:
> this is less flexible; you have discarded the error message.

Then echo it.

> the new instance would mess with the running one. 

I don't understand.  `git_maintenance_tasks_retry` is just a wrapper around `git_maintenance_tasks`.  What's the difference exactly?